business_objects/embedding.py

```diff
@@ -200,12 +201,20 @@ def get_tensors_and_attributes_for_qdrant(
             if payload_selector != "":
                 payload_selector += ","
             if data_type != enums.DataTypes.TEXT.value:
-                payload_selector += f"'{attr}', (r.\"data\"->>'{attr}')::{data_type}"
+                payload_selector += f"'{attr}', (r.\"data\"->'{attr}')::{data_type}"
```

Comment:
minor hint: the difference between -> and ->> are not clear to me. Again, just for the dumber programmers (me), the comment why the change is relevant (or even _that_ there is a change) would have been cool for me :-)

Reply:
in short previous version was faulty. -> returns a json object ->> directly casts to string. Since we cast after that anyway the second > was unnecessary from the initial introduction

Reply:
also i'm not sure if we should start explaining operators in comments because there will be no end to it